.stylelintrc.json

```diff
@@ -18,5 +18,5 @@
     },
```

Comment:
### Your Summary
![summary](https://user-images.githubusercontent.com/93039359/191243078-61171a0e-ea47-4778-8f69-0e9454cc34fc.png)

### Descriptive Summary Example :heavy_check_mark:
![summary-descriptive](https://user-images.githubusercontent.com/93039359/191243176-550fc938-97e4-4cf7-969c-ce112fd46536.png)


- [ ] You don't have a summary for your project. You can see the image above to get an idea of what a descriptive summary looks like. The summary tells in detail what are you doing in this pull request and why. :thinking: